build.gradle.kts

```diff
@@ -22,6 +24,16 @@ java {
     targetCompatibility = JavaVersion.VERSION_1_8
 }
 
+    project.plugins.withType(JavaPlugin::class) {
+        dependencies {
+            "errorprone"("com.google.errorprone:error_prone_core:2.7.1")
+            // When building ZAP releases it's also used Java 8.
+            if (JavaVersion.current() == JavaVersion.VERSION_1_8 || System.getenv("ZAP_RELEASE") != null) {
```

Comment:
The env var check is not be needed (unless you think in the near future the add-on is going to be included by default in ZAP packages).